### ISSUE TITLE
Set reasonable default values for all the losses and miners etc.

### ISSUE THREAD

**Issue by KevinMusgrave** [KevinMusgrave/pytorch-metric-learning#140]
The parameters of some losses (e.g. ArcFace) and miners (e.g. MultiSimilarityMiner) don't have default init values, or suggested values in the documentation. This makes it harder for newcomers to use.

**Comment by bobiblazeski** [comment]
This would really help. Or at least give some examples with any values, starting from blank is the hardest.

**Comment by KevinMusgrave** [comment]
I'll try to include default values in the next version 0.9.90. Later I might include explanations + maybe images of equations from the papers.

In the meantime, this table might be useful: https://kevinmusgrave.github.io/powerful-benchmarker/papers/mlrc/#optimal-hyperparameters

These hyperparameter were obtained using bayesian optimization. The setup was:

- BN-Inception model pretrained on Imagenet, with a single fully connected layer appended to get 128 dim embeddings
- RMSprop learning rate of 1e-6, and momentum 0.9. 
- Batch size 32 unless otherwise stated.

The task was to finetune the pretrained model on 3 datasets, represented by each column: CUB (200 classes of bird images), Cars (196 classes of car images), and SOP (~20000 classes of images off of Ebay).

**Comment by bobiblazeski** [comment]
Thank you for the quick reply and thank for the work on this wonderful library.

BTW I'm trying to use ArcFace  loss and it seems that  your constructor doesn't pass  the arguments required from the constructor of the base class

https://github.com/KevinMusgrave/pytorch-metric-learning/blob/master/src/pytorch_metric_learning/losses/arcface_loss.py#L11
has none of the required parameters by the LargeMarginSoftmaxLoss  https://github.com/KevinMusgrave/pytorch-metric-learning/blob/ad4b15557fb7fced2e72a1cb39564910efa04b07/src/pytorch_metric_learning/losses/large_margin_softmax_loss.py#L15 which requires: margin, num_classes, embedding_size

Unless I'm doing something wrong, the ArcFace  paper is on my TODO list

**Comment by KevinMusgrave** [comment]
They should be getting passed via ```**kwargs```

**Comment by KevinMusgrave** [comment]
All the losses and miners have default values in v0.9.90, and equations have been added to many of the losses in the documentation. See the [release notes](https://github.com/KevinMusgrave/pytorch-metric-learning/releases/tag/v0.9.90) to see breaking changes and new features.
